src/main/java/mate/model/Driver.java

```diff
@@ -47,11 +65,24 @@ public boolean equals(Object o) {
         Driver driver = (Driver) o;
         return Objects.equals(id, driver.id)
                 && Objects.equals(name, driver.name)
+                && Objects.equals(login, driver.login)
+                && Objects.equals(password, driver.password)
                 && Objects.equals(licenseNumber, driver.licenseNumber);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(id, name, licenseNumber);
+        return Objects.hash(id, name, login, password, licenseNumber);
+    }
+
+    @Override
+    public String toString() {
+        return "Driver{"
+                + "id=" + id
+                + ", name='" + name + '\''
+                + ", login='" + login + '\''
+                + ", password='" + password + '\''
```

Comment:
It's not safe to define `password` field in `toString()` method

Reply:
exactly)